tests/test_correct.py

```diff
@@ -51,18 +53,18 @@ def test_process_treats_prepif_outputs_readonly(gamma_conf, tempdir, coh_mask):
     write_config_file(params=params, output_conf_file=output_conf)
     params = Configuration(output_conf).__dict__
     conv2tif.main(params)
-    tifs = list(Path(params[C.OUT_DIR]).glob('*_unw.tif'))
+    tifs = list(Path(params[C.INTERFEROGRAM_DIR]).glob('*_unw.tif'))
     assert len(tifs) == 17
 
     if params[C.COH_FILE_LIST] is not None:
-        coh_tifs = list(Path(params[C.OUT_DIR]).glob('*_cc.tif'))
+        coh_tifs = list(Path(params[C.COHERENCE_DIR]).glob('*_cc.tif'))
         assert len(coh_tifs) == 17
 
     params = Configuration(output_conf).__dict__
     prepifg.main(params)
-    cropped_coh = list(Path(params[C.OUT_DIR]).glob('*_coh.tif'))
-    cropped_ifgs = list(Path(params[C.OUT_DIR]).glob('*_ifg.tif'))
-    dem_ifgs = list(Path(params[C.OUT_DIR]).glob('*_dem.tif'))
+    cropped_coh = list(Path(params[C.COHERENCE_DIR]).glob('*_coh.tif'))
+    cropped_ifgs = list(Path(params[C.INTERFEROGRAM_DIR]).glob('*_ifg.tif'))
+    dem_ifgs = list(Path(params[C.GEOMETRY_DIR]).glob('*_dem.tif'))
```

Comment:
Just picking this one example out, but seen it in lots of places.

You have `*_dem.tif` defined in tests here, but the expected naming of the file after `prepifg` is simply `dem.tif` - i.e. no other parts of the filename. So what are the tests comparing against?

I ran a quick test on the cropA data, which confirms the file output from `prepifg` for this PR is `geometry_dir/dem.tif`

Reply:
SB: `*_dem.tif` files are generated when we use `conv2tif`